### DISCUSSION TITLE
dart_vlc CMake issue on macos desktop

### DISCUSSION THREAD

**Discussion by hasnainelahi** [alexmercerind/dart_vlc#235] [Q&A]
Issue: it working on windows. But Getting errors on macos desktop. I need this video player in macos. How can I solve it? kindly answer it, and give me a chance to say thanks  💓

Error:
Command CompileSwiftSources failed with a nonzero exit code
CMake Error: The source directory "[/Users/hasnainelahi/Desktop/Work]() Drive[/Lab/example/macos/Pods/Drive/Lab/example/macos/Flutter/ephemeral/.symlinks/plugins/dart_vlc/macos/deps]()" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
Command PhaseScriptExecution failed with a nonzero exit code
note: Using new build system
note: Planning
note: Build preparation complete
note: Building targets in dependency order
** BUILD FAILED **

Exception: Build process failed
Exited (sigterm)

-----------------------
dependencies:
  dart_vlc: ^0.1.9
-----------------------
(cmake --version) CMake version: 3.22.3
=================
(flutter --version) Flutter: 2.10.3 • channel stable 
-----------------------
Dart: 2.16.1
-----------------------

**Comment by GroovinChip** [comment]
@hasnainelahi try regenerating your `macos` runner
